### DISCUSSION TITLE
Rpi4 with adafruit esp32 v2 WS2812

### DISCUSSION THREAD

**Discussion by monpiere** [awawa-dev/HyperSerialESP32#9] [Q&A] [answered]
I can not get any leds to work at all. I think the issue is with the data pin. How do I change to a different GPIO? Any help would be greatly appreciated. I'm so lost!

**Comment by awawa-dev** [comment, marked as answer]
Hi
I've just created wiki https://github.com/awawa-dev/HyperSerialESP32/wiki
Let me know if it's clear now.

Probably you want to change data pin/GPIO  to 4
https://github.com/awawa-dev/HyperSerialESP32/issues/5#issuecomment-1336192146
